onnx_chainer/export_testcase.py

```diff
@@ -41,20 +40,22 @@ def export_testcase(
     os.makedirs(test_data_dir, exist_ok=True)
     for i, var in enumerate(inputs):
         with open(os.path.join(test_data_dir, 'input_%d.pb' % i), 'wb') as f:
-            t = numpy_helper.from_array(var.data, 'Input_%d' % i)
+            array = chainer.cuda.to_cpu(var.array)
+            t = numpy_helper.from_array(array, 'Input_%d' % i)
             f.write(t.SerializeToString())
 
     for i, var in enumerate(outputs):
         with open(os.path.join(test_data_dir, 'output_%d.pb' % i), 'wb') as f:
-            t = numpy_helper.from_array(var.data, '')
+            array = chainer.cuda.to_cpu(var.array)
+            t = numpy_helper.from_array(array, '')
```

Comment:
Maybe better to set actual output names. TODO would be fine (I think we would need to refactor code around here when we support custom input/output names).

Reply:
add TODO